### ISSUE TITLE
Do you have plans to implement yolov7-pose

### ISSUE THREAD

**Issue by APeiZou** [open-mmlab/mmyolo#576]
### Model/Dataset description

@open    Do you have plans to implement yolov7-pose

### Open source status

- [X] The model implementation is available
- [X] The model weights are available.

**Comment by hhaAndroid** [comment]
@APeiZou Yes, but time is really limited. If there is a community willing to help that would be great.

**Comment by APeiZou** [comment]
@hhaAndroid Thank you for your reply.
